tests/solvers/test_sundials.py

```diff
@@ -874,6 +782,106 @@ def test_rtol_vector_sense(self):
         sim.rtol = [1e-6]
         sim.rtol = np.array([1e-6])
 
+    def test_no_progress_force_min_h(self):
+        """ Test example where CVode fails to make progress and minimal stepsize 
```

Comment:
Just a note some of the added test docstrings start as `""" x.""""` and others as `"""x."""`

Reply:
Made it a bit more uniform.